include/h2o/mruby.h

```diff
@@ -0,0 +1,31 @@
+#ifndef H20_MRUBY_H                                                              
+#define H20_MRUBY_H                                                              
+
+#include "h2o.h"
+#include <mruby.h>
+
+#define MODULE_NAME "h2o_mruby"
+#define MODULE_VERSION "0.0.1"
+#define MODULE_DESCRIPTION MODULE_NAME "/" MODULE_VERSION
```

Comment:
Names defined in public headers must be prefixed with `h2o_` (or in this case `h2o_mruby_`).
Please either prefix the names, or move the definitions to somewhere within the source files.